FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -795,6 +795,224 @@ paths:
+      description: |
+        Retrieve the unavailability schedule for a specific component.
+        - The response will group days with similar time periods together.
+        - If a day has a unique unavailability period that does not match any other day, it will be listed separately.
+        - If a day has no unavailability scheduled, it will be included with an empty `timePeriods` array.
+      operationId: getUnavailabilityScheduleUsingGET
+      parameters:
+      - in: path
+        name: uuid
+        description: The UUID of the component
+        required: true
+        schema:
+          type: string
+          format: uuid
+      responses:
+        200:
+          description: Successfully retrieved the unavailability schedule
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnavailabilitySchedule'
+        401:
+          description: Not authenticated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden - client is not authorized to read the schedule
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        406:
+          description: Not acceptable - only responds in application/json format
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        500:
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - OAuth2: [fraihmwork.component.read, fraihmwork.component.write, fraihmwork.component.admin]
+
+    delete:
+      tags:
+      - Components
+      summary: Delete the unavailability schedule for a component.
+      description: |
+        Delete the unavailability schedule for a component.
+        - If only the component ID is provided, all days and all time periods will be deleted.
+        - If a specific day is provided, the entire unavailability for that day, including all its time periods, will be deleted.
+        - If both a day and a time period are specified, only that specific time period for the given day will be deleted.
+      operationId: deleteUnavailabilityScheduleUsingDELETEQuery
+      parameters:
+      - in: path
+        name: uuid
+        description: The UUID of the component
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - in: query
+        name: day
+        description: The day of the week to delete schedules for
+        required: false
+        schema:
+          type: string
```

Comment:
If you are specifying this array in multiple places, it should be extracted out as its own thing

Reply:
agreed on that, since decided to move from delete api to post api using /removed endpoint be using the same schema for the requestBody instead of query parameters therefore separate schema is no longer needed